main.go

```diff
@@ -441,6 +441,10 @@ func main() {
 					Name:  "include-transitive-needs",
 					Usage: `like --include-needs, but also includes transitive needs (needs of needs). Does nothing when when --selector/-l flag is not provided. Overrides exclusions of other selectors and conditions.`,
 				},
+				cli.BoolFlag{
+					Name:  "validate",
+					Usage: "validate your manifests against the Kubernetes cluster you are currently pointing at. Note that this requiers access to a Kubernetes cluster to obtain information necessary for validating, like the list of available API versions",
```

Comment:
```suggestion
					Usage: "ADVANCED CONFIGURATION: When sync is going to involve helm-template as a part of the "chartify" process, it might fail due to missing .Capabilities. This flag makes instructs helmfile to pass --validate to helm-template so it populates .Capabilities and validates your manifests against the Kubernetes cluster you are currently pointing at. Note that this requiers access to a Kubernetes cluster to obtain information necessary for validating, like the list of available API versions",
```

Maybe we'd better be more explicit about when to use this, or just default to pass `--validate` when `sync`, as when one is about to helmfile-sync, it might be safe to say that it is expected to access the cluster anyway.

Reply:
@mumoshu 
> Or, should we just default to pass `--validate` when `sync`
I'm not sure it's necessary to default pass this to `sync`

In my opinion if we decide to set `--validate` as default we must do it onto `template` as this Flags is only available for templating in Helm.
For now i'd prefer defining this flag when needed

Reply:
@chubchubsancho `helmfile-template` is designed to (usually) run locally without cluster access, whereas `helmfile-sync` is not. That's why I thought it would be a good idea to make the validate flag a default only for `helmfile-sync`.

Anyway, this enhancement is very useful on its own! Let's merge and see how folks use it.